### PR TITLE
Fixing a few grammar and punctuation mistakes on README.md

### PR DESCRIPTION
**Pull request by cs-student101**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+[Kitodo](https://github.com/kitodo) is an open source software suite intended to support mass digitization projects for cultural heritage institutions. Kitodo is widely used and cooperatively maintained by major German libraries and digitization service providers. The software implements international standards such as METS, MODS, ALTO, and other formats maintained by the Library of Congress. Kitodo consists of several independent modules serving different purposes such as controlling the digitization workflow, enriching descriptive and structural metadata, and presenting the results to the public in a modern and convenient way.
 
-To get more information, visit the [Kitodo homepage](https://www.kitodo.org). You can also follow Kitodo News on [Twitter](https://twitter.com/kitodo_org).
+For get more information, visit the [Kitodo homepage](https://www.kitodo.org). You can also follow Kitodo News on [Twitter](https://twitter.com/kitodo_org).
```
Comment (solth) [inline README.md]: I guess this should be `For more information...`?

Comment (cs-student101) [reply README.md]: Yes, it should be "For more information...", sorry about the typo.

Comment (Kathrin-Huber) [reply README.md]: Are you going to change this?

File: README.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
 Kitodo.Production
 =================
 
-Kitodo.Production is a workflow management tool for mass digitization.  It is part of the Kitodo Digital Library Suite.
+Kitodo.Production is a workflow management tool for mass digitization and is part of the Kitodo Digital Library Suite.
 
-Kitodo.Production supports various types of materials such as prints, periodicals, manuscripts, sheet music and typical documents of a *Nachlass*. The software is written in Java and using Java Server Faces web technology to run on a [Tomcat Servlet container](http://tomcat.apache.org/) backed up by a [MySQL](http://www.mysql.com) database accessed utilizing the [Hibernate framework](http://www.hibernate.org).
+Kitodo.Production supports various types of materials such as prints, periodicals, manuscripts, sheet music and typical documents of a *Nachlass*. The software is written in Java, uses Java Server Faces web technology to run on a [Tomcat Servlet container](http://tomcat.apache.org/), and backed by a [MySQL](http://www.mysql.com) database utilizing the [Hibernate framework](http://www.hibernate.org) to access it.
```
Comment (Kathrin-Huber) [inline README.md]: It should be "and is backed by"

File: README.md (hunk 1 above)
Comment (cs-student101) [reply README.md]: Yep, changed just now. Sorry, forgot about that.

File: README.md (hunk 2 above)
Comment (cs-student101) [reply README.md]: Sorry about the typo, changed just now.

**Pull request merged**